UI/Profile.tsx

```diff
@@ -43,32 +45,75 @@ export const ProfileScreen = () => {
   });
 
   const onEdit = async () => {
-    try {
-      await fetch(editProfileAPI, {
-        method: 'POST',
-        body: body,
-        headers: {
-          'Content-Type': 'application/json',
-          // eslint-disable-next-line prettier/prettier
-          'Authorization': tk,
-        },
-      });
-    } catch (error) {
-      console.error('ERROR: edit profile');
-    } finally {
-      setMessage('User profile updated.');
-      await AsyncStorage.setItem('@firstName', newFirstName);
-      await AsyncStorage.setItem('@lastName', newLastName);
-      await AsyncStorage.setItem('@password', newPassword);
+    setMessage('');
+    setErrorName('');
+    const validateInputs = validateAll();
+    if (validateInputs === true) {
+      console.log(body);
```

Comment:
Now that the app is complete, let's take the console logs (except the error ones) out to improve performance.